test/fixtures/test-runner/global-setup-teardown/basic-setup-teardown.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
```

Comment:
Here and in other places:

```suggestion
const fs = require('node:fs');
```